x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/job/RollupIndexer.java

```diff
@@ -58,7 +58,7 @@
  * An abstract implementation of {@link AsyncTwoPhaseIndexer} that builds a rollup index incrementally.
  */
 public abstract class RollupIndexer extends AsyncTwoPhaseIndexer<Map<String, Object>, RollupIndexerJobStats> {
-    static final String AGGREGATION_NAME = RollupField.NAME;
+    static final String AGGREGATION_NAME = RollupField.NAME.replaceAll("[\\[\\]\\>]","");
```

Comment:
Could you clarify what this line is doing?

Reply:
This is the part I wasn't so sure about. Honestly, I am still a bit confused on where I should put this, 
but my intention was to eliminate the invalid characters `[`, `]`, `>` from the RollUpField name to prevent
side affects.

Reply:
`RollupField.NAME` is a constant:  https://github.com/elastic/elasticsearch/blob/63012c8a40742829806b44cb6577b05d9ab25a83/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/RollupField.java#L37 

I don't think any invalid characters can show up there.

Reply:
Sorry, I don't know what I was thinking there. I'l try to find where they could show up.